workflows/prognostic_scream_run/tests/test_scream_config.py

```diff
@@ -0,0 +1,48 @@
+import os
+import scream_run
+import yaml
+import pytest
+from typing import List
+
+dirname = os.path.dirname(os.path.abspath(__file__))
+
+EXAMPLE_CONFIGS_DIR = os.path.join(dirname, "example_configs")
+
+
+@pytest.mark.parametrize(
+    "path, file", [(EXAMPLE_CONFIGS_DIR, "scream_ne30pg2.yaml")],
+)
+def test_example_config_can_initialize(path: str, file: str):
+    with open(os.path.join(path, file), "r") as f:
+        config = scream_run.ScreamConfig.from_dict(yaml.safe_load(f))
+    assert isinstance(config, scream_run.ScreamConfig)
+
+
+output_bucket_dir = "gs://vcm-scream/config/output"
+
+
+@pytest.mark.parametrize(
+    "path, filename", [pytest.param(output_bucket_dir, "default.yaml"),],
+)
+def test_resolve_single_output_yaml(path: str, filename: str):
+    config = scream_run.ScreamConfig()
+    config.output_yaml = os.path.join(path, filename)
+    config.resolve_output_yaml("/tmp")
```

Comment:
Here and in the other test, should you use the [`tmp_path` fixture of pytest](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) to handle creating and cleaning up these temporary files more cleanly?

Reply:
Great idea, changed to pytest fixture.